### ISSUE TITLE
How to use different view points to test the model?

### ISSUE THREAD

**Issue by yys123456** [saeedhd96/neural-radiosity#2]

**Comment by yys123456** [comment]
I tested the cornell box scene, but only got 1 image called `000.png` in `lhs` or `rhs` folder. And I print the cfg in `common.py`, I got `n_views = -1` and `cameras = None`, and I don't know how to add more cameras.

**Comment by saeedhd96** [comment]
Hello,

Please note the word 'test' view might be misleading, becuase there is no training view used. This method is view-independent, and the training process is also view independent, meaining that we do not use any pre-rendered ground truth input images to train our model. Therefore, after training, any view is a 'test' view.

Regarding adding more cameras, please check this new [sample script](https://github.com/saeedhd96/neural-radiosity/blob/master/sample_scripts/all_scenes/test_nerad_multiview.sh) I just added. You would need to replace your cameras.xml with the file attached below.

[cameras.xml.zip](https://github.com/saeedhd96/neural-radiosity/files/12149480/cameras.xml.zip)

Please let me know if this helped.

**Comment by yys123456** [comment]
Thank you very much for the information, I got a clearer understanding of the model. And for the multiple cameras I actually figured out by myself yesterday🙂.

**Comment by yys123456** [comment]
@saeedhd96 Does Neural Radiosity support environment maps?

**Comment by saeedhd96** [comment]
Yes. Lighting form has nothing to do with Neural Radiosity. Neural Radiosity just learns the radiance distribution under any lighting. It is completely orthogonal. In face, we have already showed experiments on the NeRF scenes (lego etc.) with Inverese Neural Radiosity where the lighting was envamp: https://inverse-neural-radiosity.github.io
